src/test/resources/Login.feature

```diff
@@ -1,12 +1,23 @@
-Feature: Login
-
-  An a user
-  I want to sign in
-  So I can use the app
-
-  @automated
-  Scenario: User can sign in with valid value
-    Given user is in Katalon login page1
-    When user sets username as "John Doe" and password as "ThisIsNotAPassword"
-    And user clicks on login button
-    Then user sees appointment page
+Feature: Login
+
+  As an user
+  I want to sign in
+  So I can use the app
+
+  @automated
+  Scenario: User can sign in with valid value
+    Given user is in Katalon login page
+    When user sets username as "John Doe" and password as "ThisIsNotAPassword"
+    And user clicks on login button
+    Then user sees appointment page
+
+  Scenario Outline: User can sign in with invalid value
+    Given user is in Katalon login page
+    When user sets username as "<username>" and password as "<password>"
+    And user clicks on login button
+    Then user sees error message as "<error_message>"
```

Comment:
có thể xóa cột: `error_message`
vì cùng hiển thị 1 error nên e có thể viết như này:
user sees error message as "Login failed! Please ensure the username and password are valid."